docs/api/api_docs.yaml

```
@@ -0,0 +1,1189 @@
+openapi: 3.0.0
+info:
+  title: Sync Gateway API
+  version: '3.0'
+  license:
+    name: Business Source License 1.1 (BSL)
+    url: 'https://github.com/couchbase/sync_gateway/blob/master/LICENSE'
+  description: ''
+servers:
+  - url: 'http://localhost:4984'
+    description: Public API
+  - url: 'http://localhost:4985'
+    description: Admin API
+  - url: 'http://localhost:4986'
+    description: Metric API
+paths:
+  /:
+    get:
+      summary: Abc
+      responses: {}
```

Just fill in a default value containing the required fields to get the spec validating.

```suggestion
      responses: 
        default: 
          description: TODO
```